components/rmrk/Gallery/Search/SearchBar.vue

```diff
@@ -145,3 +344,15 @@ export default class SearchBar extends Vue {
   opacity: 0;
 }
 </style>
+
+<style lang="scss">
+.image img {
+    width: 32px ;
+    height: 32px ;
+}
+.history {
+  color: #d32e79;
+  font-size: 15px;
+}
+
+</style>
```

Comment:
This will be global, it should be moved to scopped scss

Reply:
@roiLeo If I move .image part to scoped scss, it doesn't work. Only .history part works inside scoped scss.

Reply:
why do you need `.image img` rule? you have already set up size with `is-32x32`

Reply:
You should rename `history` to something more specific. And be sure to replace color hex code with scss variable

Reply:
`is-32x32` make the size smaller but some images appear rectangular. So `.image img` makes every image square

Reply:
> `is-32x32` make the size smaller but some images appear rectangular. So `.image img` makes every image square

Have you tried using `<BasicImage />` component?

Reply:
Yeah I tried that first. But it uses ratio and it seems ratio and size don't work well together.

Reply:
something like this should work
```js
<BasicImage
  customClass="is-32x32"
  :src="props.option.image === '' ? props.option.animation_url : props.option.image"
/>
```

Reply:
The size seems to change the width but ratio keeps the original height. So only a part of the image is visible.

Reply:
This doesn't seem to be a problem to me as we use the same ratio for Gallery Card list

Reply:
> something like this should work
> 
> ```
> <BasicImage
>   customClass="is-32x32"
>   :src="props.option.image === '' ? props.option.animation_url : props.option.image"
> />
> ```

Oh yeah, it worked. But checkout what happens when we add the fallback image.

https://user-images.githubusercontent.com/44033666/147103571-0269fb40-6e35-44a0-abfd-1145f3a37cd5.mp4

Once the fallback image is loaded, then some of the original images take time to reload. Can you figure out what the issue might be?

Reply:
This bug?
![Screenshot 2021-12-22 at 15-42-04 nft-gallery-nuxt](https://user-images.githubusercontent.com/9987732/147109846-08765392-3888-49d3-b2cb-70798b4dd775.png)

Reply:
Hmm, somehow `<BasicImage>` error function is called and the result neither has an image field nor an animation_url. So, instead of an image some text is printed. When does `<BasicImage>` run the error function?

Reply:
Printed text come from image `alt` attribute, it could be hidden but I don't know if it's the right solution.
We need to investigate why images can't load some times.

Reply:
Fixed that by loading the result in async manner.

Reply:
But discovered a new bug in `b-autocomplete`. Once you move the arrow keys then that element gets selected. If you type new keyword, still the old result would be selected as the position is not reset. Check out in the video how `spades 9` gets selected even when it's not selected.

https://user-images.githubusercontent.com/44033666/147212653-245b2f81-d3ac-4a11-8692-c3bb06ccaccc.mp4